variables.tf

```diff
@@ -15,3 +15,7 @@ variable "CLIENTSECRET" {
     type        = string
     sensitive   = true
 }
+variable "loglevel" {
+    description = "level for the logs."
+    default = "ERROR"
```

Comment:
Is this the best default? I'd probably want at least warning logs in my test/production apps, and potentially Info too